tests/features/eap/eap_gc.feature

```diff
@@ -90,3 +90,28 @@ Feature: Openshift OpenJDK GC tests
       | env_json  | {"INITIAL_HEAP_PERCENT": 0.5} |
     Then container log should match regex ^ *JAVA_OPTS: *.* -Xms256m\s
       And container log should match regex ^ *JAVA_OPTS: *.* -Xmx512m\s
+
+  Scenario: CLOUD-1524, test JAVA_CORE_LIMIT
+    When container is started with env
+      | variable              | value    |
+      | JAVA_CORE_LIMIT       | 3        |
+    Then container log should match regex ^ *JAVA_OPTS: *.* -XX:ParallelGCThreads=3\s
+     And container log should match regex ^ *JAVA_OPTS: *.* -Djava.util.concurrent.ForkJoinPool.common.parallelism=3\s
+     And container log should match regex ^ *JAVA_OPTS: *.* -XX:CICompilerCount=2\s
+
+  Scenario: CLOUD-1524, test JAVA_CORE_LIMIT and CONTAINER_CORE_LIMIT
+    When container is started with env
+      | variable             | value    |
+      | JAVA_CORE_LIMIT      | 3        |
+      | CONTAINER_CORE_LIMIT | 6        |
+    Then container log should match regex ^ *JAVA_OPTS: *.* -XX:ParallelGCThreads=6\s
```

Comment:
IMHO the behavior should the opposite  (JAVA_CORE_LIMIT should take precedence, as CONTAINER_CORE_LIMIT is set from the actual cpu limit)